src/lazy.js

```diff
@@ -137,7 +137,7 @@ export default function (Vue) {
             const exist = find(this.ListenerQueue, item => item.el === el)
 
             exist && exist.src !== src && exist.update({
-                src,
+                src: exist.src,
```

Comment:
src is declared from `let { src, loading, error } = this.valueFormatter(binding.value)`， that's right.